src/Crypto/WebAuthn/Operation/CredentialEntry.hs

```diff
@@ -16,7 +16,6 @@ import GHC.Generics (Generic)
 -- store for each credential that's registered to a user
 data CredentialEntry = CredentialEntry
   { ceCredentialId :: M.CredentialId,
-    ceUserHandle :: M.UserHandle,
```

Comment:
Whenever we have a `CredentialEntry` we already _know_ the `UserHandle`.
As we look up the credential entry with a function `lookup :: UserHandle -> CredentialId -> Maybe CredentialEntry` now.